src/Bridges/ApplicationDI/ApplicationExtension.php

```diff
@@ -65,4 +75,67 @@ public function loadConfiguration()
 		}
 	}
 
+
+	public function beforeCompile()
+	{
+		$config = $this->validateConfig($this->defaults);
+		$container = $this->getContainerBuilder();
+		$counter = 0;
+
+		foreach ($this->findClasses() as $class) {
+			if ($services = $container->findByType($class, FALSE)) {
```

Comment:
This should search by tag as well.
And before processing found classes, there should be one more pass over already defined presenter services, that would add appropriate tags to them.